tests/Http/Event/UpdateSubEventsRequestHandlerTest.php

```diff
@@ -0,0 +1,287 @@
+                        ->withStatus(new Status(StatusType::available(), []))
+                ),
+            ],
+            'one_subEvent_with_id_and_status_type_and_reason' => [
+                'data' => [
+                    (object) [
+                        'id' => 1,
+                        'status' => (object) [
+                            'type' => 'Unavailable',
+                            'reason' => (object) [
+                                'nl' => 'Geannuleerd wegens covid',
+                                'fr' => 'Franse tekst',
+                            ],
+                        ],
+                    ],
+                ],
+                'expected_command' => new UpdateSubEvents(
+                    self::EVENT_ID,
+                    (new SubEventUpdate(1))
+                        ->withStatus(
+                            new Status(
+                                StatusType::unavailable(),
+                                [
+                                    new StatusReason(new Language('nl'), 'Geannuleerd wegens covid'),
+                                    new StatusReason(new Language('fr'), 'Franse tekst'),
+                                ]
+                            )
+                        ),
+                ),
+            ],
+            'one_subEvent_with_id_and_bookingAvailability_type' => [
+                'data' => [
+                    (object) [
+                        'id' => 1,
+                        'bookingAvailability' => (object) [
+                            'type' => 'Unavailable',
+                        ],
+                    ],
+                ],
+                'expected_command' => new UpdateSubEvents(
+                    self::EVENT_ID,
+                    (new SubEventUpdate(1))
+                        ->withBookingAvailability(BookingAvailability::unavailable()),
+                ),
+            ],
+            'one_subEvent_with_id_and_status_type_and_bookingAvailability_type' => [
+                'data' => [
+                    (object) [
+                        'id' => 1,
+                        'status' => (object) [
+                            'type' => 'Available',
+                        ],
+                        'bookingAvailability' => (object) [
+                            'type' => 'Unavailable',
+                        ],
+                    ],
+                ],
+                'expected_command' => new UpdateSubEvents(
+                    self::EVENT_ID,
+                    (new SubEventUpdate(1))
+                        ->withStatus(new Status(StatusType::available(), []))
+                        ->withBookingAvailability(BookingAvailability::unavailable()),
+                ),
+            ],
+        ];
+    }
+
+    /**
+     * @test
+     * @dataProvider invalidDataProvider
+     */
+    public function it_throws_an_api_problem_when_given_with_valid_data($data, array $expectedSchemaErrors): void
```

Comment:
I think it should be `invalid_data` instead of `valid_data`

Reply:
Good find 🙈 Fixed in 9e8c390